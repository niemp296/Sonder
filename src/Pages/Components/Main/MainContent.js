import React from 'react';
//import SideBar from '../SideBar/SideBar'
import SearchBar from '../SearchBar/SearchBar'
import './Main.css'

function Main(props) {
    return (
      <div className="Main">
        <SearchBar isLoggedIn ={props.isLoggedIn}/>
      </div>
    );
  }
  
  export default Main;
  