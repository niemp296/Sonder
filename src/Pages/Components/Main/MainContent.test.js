import React from 'react';
import { mount, shallow , configure} from 'enzyme';
import MainContent from './MainContent';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});

describe("Test case for Main Content layout", () => {
    it("should render main content page", () => {
        const wrapper = shallow(<MainContent />);
    });

    it("should render initial layout", () => {
        // when
        const component = shallow(<MainContent/>);
        // then
        expect(component.getElements()).toMatchSnapshot();
    });
});


