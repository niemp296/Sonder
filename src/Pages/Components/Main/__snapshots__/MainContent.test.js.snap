// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test case for Main Content layout should render initial layout 1`] = `
Array [
  <div
    className="Main"
  >
    <SearchBar />
  </div>,
]
`;
